projects/packages/jitm/src/class-jitm.php

```diff
@@ -115,7 +115,7 @@ public function jitm_enqueue_files() {
 
 		wp_enqueue_script(
 			'jetpack-jitm-new',
-			Assets::get_file_url_for_environment( '_inc/build/jetpack-jitm.min.js', '_inc/jetpack-jitm.js' ),
+			Assets::get_file_url_for_environment( 'assets/js/jetpack-jitm.min.js', 'assets/js/jetpack-jitm.js', __DIR__ ),
```

Comment:
In the Lazy Images package, we opted to have the assets inside `src/`, and here we have them in their own `assets/` dir. Would it make sense to move everything inside of `src/` now?

Reply:
In commit 803c759, I moved the css files to `src/css` and the js file to `src/js`.